src/lib/handlebars.js

```diff
@@ -86,5 +86,23 @@ module.exports = () => {
 		))
 	})
 
+	handlebars.registerHelper('getPowerUpCost', (levelStart, levelEnd) => {
+		if (!levelStart || !levelEnd) return ''
+		let stardustCost = 0
+		let candyCost = 0
+		let xlCandyCost = 0
+		for (const level in powerUpCost) {
+			if (level >= levelStart && level < levelEnd) {
+				stardustCost = stardustCost + powerUpCost[level].stardust
+				if (powerUpCost[level].candy) candyCost = candyCost + powerUpCost[level].candy
+				if (powerUpCost[level].xlCandy) xlCandyCost = xlCandyCost + powerUpCost[level].xlCandy
+			}
+		}
+		let returnString = `${stardustCost} ${translator.translate('Stardust')}`
```

Comment:
```suggestion
		let returnString = `${stardustCost.toLocaleString()} ${translator.translate('Stardust')}`
```

Adds comma to the stardust amount